explore/src/main/scala/explore/SideTabs.scala

```diff
@@ -23,35 +24,46 @@ import react.semanticui.elements.divider.Divider
 import react.semanticui.sizes._
 import react.semanticui.widths._
 
-final case class SideTabs[P](
-  router: RouterCtl[P],
-  tabs:   Zipper[SideButton]
-) extends ReactProps[SideTabs[_]](SideTabs.component)
+final case class SideTabs(
+  tabs: View[Zipper[AppTab]]
+) extends ReactProps[SideTabs](SideTabs.component)
 
 object SideTabs {
-  type Props = SideTabs[_]
+  type Props = SideTabs
 
-  implicit val reuse: Reusability[Props] = Reusability.by(_.tabs)
+  implicit val propsReuse: Reusability[Props] = Reusability.derive
 
-  private val component =
+  private def componentBuilder[A] =
     ScalaComponent
       .builder[Props]
       .stateless
       .render_P { p =>
-        val tabsL = p.tabs.toNel
-        val focus = p.tabs.focus
+        val tabsL = p.tabs.get.toNel
+        val focus = p.tabs.get.focus
+
+        def tabButton(tab: AppTab): Button =
+          Button(positive = tab === focus,
+                 onClick = p.tabs.mod(z => z.findFocus(_ === tab).getOrElse(z)).runInCB
+          )(tab.title)
+
         <.div(
           GPPStyles.SideTabsBody,
-          VerticalSection()(Button(active = tabsL.head === focus)(tabsL.head.title)),
+          VerticalSection()(
+            tabButton(tabsL.head)
+          ),
           Divider(hidden = true),
           VerticalSection()(
             ButtonGroup(widths = widthOf(tabsL.length))(
               // Due to the css rotations these need to be in reversed order
-              tabsL.tail.reverse.map(b => Button(active = b === focus)(b.title)).toTagMod
+              tabsL.tail.reverse
+                .map(tabButton)
+                .toTagMod
             )
           )
         )
       }
       .configure(Reusability.shouldComponentUpdate)
       .build
+
+  val component = componentBuilder[Any]
```

Comment:
Yes, that's the mechanism we are using in other parts but it should actually be removed from here. `SideTabs` was type parametrized (on the `Page`) but since we removed the router from the props, it doesn't have type parameters anymore.